### PR TITLE
Adding description to cs proj file

### PR DESCRIPTION
**Pull request by aishwaryabh**
Adding description to csproj file for publishing nuget package

### CODE REVIEW

File: src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <Description>Microsoft Azure Functions Worker OpenAI Extension</Description>
```
Comment (manvkaur) [inline src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj:5]: should we mention for .NET isolated functions

File: src/Directory.Build.props [hunk 2]
```diff
@@ -18,6 +18,7 @@
     <Authors>Microsoft</Authors>
     <Company>Microsoft</Company>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <Description>Microsoft Azure WebJobs SDK OpenAI Extension</Description>
```
Comment (manvkaur) [inline src/Directory.Build.props:21]: let's keep it in csproj files only

File: src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj (hunk 1 above)
Comment (cgillum) [reply src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj:5]: +1. I suggest "Azure Functions .NET Isolated worker extension for OpenAI"

File: src/Directory.Build.props (hunk 2 above)
Comment (cgillum) [reply src/Directory.Build.props:21]: +1

**Review by manvkaur — APPROVED** [review]

**Pull request merged**